model-optimizer/mo/middle/passes/mean_scale_values.py

```diff
@@ -20,62 +20,75 @@
 from mo.middle.pattern_match import apply_pattern
 
 
-def move_scaleshift_to_preprocess_action(graph, match):
+def move_mean_values_to_preprocess_action(graph, match):
     mean_values = {}
     input_op = match['input_op']
-    scale_shift = match['scale_shift']
-    weights = np.squeeze(match['weights'].value)
-    biases = np.squeeze(match['biases'].value)
+    add_op = match['add']
+    mul_op = match['mul']
 
-    if graph.graph['cmd_params'].reverse_input_channels:
-        biases = np.flip(biases)
+    if match['const_mul_output'].has('value'):
+        weights = match['const_mul_output'].value
+        if any([x != 1 for x in weights]):
+            return
 
-    if any([x != 1 for x in weights]):
-        return
+    # # Keep biases (mean values) for current input as graph attr and remove mean values layers
+    # # Input->data->Mul->Add->scsh_data    =>    Input->scsh_data
+    add_op.out_port(0).get_connection().set_source(input_op.out_port(0))
+    mul_op.in_port(0).disconnect()
```

Comment:
We can just remove the `mul_op` and corresponding data node. The same commend for `add_op`